### PR TITLE
Expand introduction slides

### LINKED ISSUE #324: More expansive Introduction lecture
As discussed, we should start the workshop with a proper introduction to the core concepts / ideas of the workshop and motivation for all the parts, make it easier for students to see "the big picture".

Rough outline for dicussion:

* Basic Goals
  * Teach everyone basics in programming → Python
  * Basic data analysis for the lab courses → numpy, matplotlib, fitting (scipy or iminuit?)
  * Scientific Writing → LaTeX

* Open Science 
   * Free and Open Source Software → Everything

* Reproducible Science
  * Reproducibility Crisis
  * Version Control → git
  * Workflow Automatization → (snake-?) make

* The different operating systems (aka windows sucks)
* Text Editors

### PR DESCRIPTION
**Pull request by maxnoe**
Fixes #324

Missing: Discussion of Operating Systems

### CODE REVIEW

**Review by aknierim — CHANGES_REQUESTED** [review]
Looks good, but we may want to emphasize the importance of the second week, since that is the part that historically shows the greatest decline in participants due to the holiday... So maybe on slide 22 (Ablauf) something like "Nächste Woche: Protokolle schreiben mit LaTeX"?

File: intro/content/intro.tex [hunk 1]
```diff
@@ -1,33 +1,133 @@
...
 
-\begin{frame}{Ziele}
+
+\begin{frame}{Motivation}
   \begin{center}
-    \textcolor{vertexDarkRed}{\Huge Auf das Praktikum vorbereiten} \\[3ex]
-    \Large Daten auswerten \hspace{2em} Plotten \hspace{2em} Fehlerrechnung
+    \textcolor{vertexDarkRed}{\Huge Auf das Praktikum vorbereiten} \\[\baselineskip]
+    \Large Daten:\hspace{2em} Abspeichern \hspace{2em} Auswerten \hspace{2em} Visualisieren \\ [\baselineskip]
+    Unsicherheiten bestimmen \hspace{2em} Protokoll verfassen\\
```
Comment (chrbeckm) [inline intro/content/intro.tex:8]: I would change the second line to:
Zusammen arbeiten \hspace{2em} Protokoll verfassen
For me, uncertainties are part of the Auswertungs-part.

File: intro/content/intro.tex [hunk 2]
```diff
@@ -1,33 +1,133 @@
...
   \end{center}
 \end{frame}
 
-\begin{frame}{Ziele}
+\begin{frame}{Motivation}
   \begin{center}
-    \textcolor{vertexDarkRed}{\Huge Von Anfang an: Best Practices} \\[3ex]
+    \textcolor{vertexDarkRed}{\Huge Von Anfang an: Best Practices} \\[\baselineskip]
     \Large%
     Spart Zeit und Nerven\hspace{2em}
```
Comment (chrbeckm) [inline intro/content/intro.tex:28]: baselineskip in place of hspace

File: intro/content/intro.tex [hunk 3]
```diff
@@ -1,33 +1,133 @@
...
+      \item Gründe sind vielfältig, aber hauptsächlich
+        \begin{itemize}
+          \item Publikations-Bias (positive Ergebnisse werden veröffentlicht, Null-Resultate nicht)
+          \item Zugrunde liegende Daten nicht verfügbar
+          \item Genutzte Software nicht verfügbar
+          \item Mangelhafte statistische Analyse
+          \item Veröffentlichungen mit der vollen Information hinter \enquote{Pay Walls}
+          \item ...
+        \end{itemize}
+      \item Das muss besser gehen...
```
Comment (chrbeckm) [inline intro/content/intro.tex:52]: gehen!

File: intro/content/intro.tex [hunk 4]
```diff
@@ -1,33 +1,133 @@
...
+    \textcolor{vertexDarkRed}{\Huge FAIR Prinzipien für Daten} \\[\baselineskip]
+
+    \begin{tblr}{}
+      F & Findable \\
+      A & Accessible  \\
+      I & Interoperable \\
+      R & Reusable \\
+    \end{tblr}
+
+    Kurz: Wissenschaftliche Daten müssen in wohldefinierten, offenen Datenformaten mit
```
Comment (chrbeckm) [inline intro/content/intro.tex:97]: beschreiben, abgespeichert werden.

**Review by chrbeckm — CHANGES_REQUESTED** [review]

File: intro/content/intro.tex (hunk 1 above)
Comment (aknierim) [reply intro/content/intro.tex:8]: Just a slight nuance, but "Zusammenarbeiten" would be better, as it implies working together on a project (e.g. to finish it together), whereas "zusammen arbeiten" only implies working together. The latter does not necessarily mean working on the same project.

**Review by aknierim — APPROVED** [review]

**Review by chrbeckm — APPROVED** [review]

**Comment by maxnoe** [comment]
> Missing: Discussion of Operating Systems

Will add in another PR, also a general discussion of what open source is is missing

**Pull request merged**